File: estoque/src/main/java/com/example/estoque/model/Fornecedor.java
package com.example.estoque.model;

import lombok.Data;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import java.util.UUID;

@Data
@Entity
@Table(name = "fornecedor")
public class Fornecedor {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;

    private String nome;

    @Column(name = "codigo_pais", nullable = false)
    private String codigoPais;
}

File: estoque-web/src/model/fornecedor.ts
export class Fornecedor {
    id: string;
    nome: string;
    codigoPais: string;
    nomePais: string;
    bandeiraPais: string;

    constructor(
        id: string,
        nome: string,
        codigoPais: string,
        nomePais: string,
        bandeiraPais: string,
    ) {
        this.id = id;
        this.nome = nome;
        this.codigoPais = codigoPais;
        this.nomePais = nomePais;
        this.bandeiraPais = bandeiraPais;
    }
}